infra/license/package-judgment.json

```diff
@@ -76,5 +76,11 @@
       "licenses": "(MIT OR CC0-1.0)",
       "repository": "https://github.com/sindresorhus/type-fest",
       "permitted": "yes"
+    },
+
+    "flatbuffers@2.0.7": {
```

Comment:
I missed this :(
This should be alphabetic order!
Please move this to the right place :D